src/components/formRow/formRow.scss

```diff
@@ -105,6 +105,10 @@
     padding-right: 25px;
   }
 
+  input[type=date] {
```

Comment:
Why do you need to specify `font-family` to date inputes?

Reply:
otherwise, only for date component, an ugly font is used https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date. 

I'm open to another solution but this was the quickest one.

Reply:
Hmm ok. So could you at least use the same font the rest of the app is using?
`font-family: 'Open Sans', sans-serif;`